packages/terra-consumer-nav/src/components/nav-help/NavHelp.scss

```diff
@@ -0,0 +1,105 @@
+@import '../../variables';
+
+:local {
+  .nav-help {
+    background-color: var(--terra-consumer--help-button-background, #fff);
+    border: 0;
+    border-radius: 20px;
+    box-shadow: 0 1px 1px #666;
+    color: var(--terra-consumer--help-button-color, #3d3d3d);
+    display: inline-block;
+    font-family: inherit;
+    font-size: 16px;
```

Comment:
this may be an enhancement for later, but what do you think about making all these font-size's and line-heights either a single variable, or multiple variables?